docs/_static/cookie_notice.js

```diff
@@ -0,0 +1,29 @@
+const send_ga = () => {
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+   (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+   m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+   })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
```

Comment:
I don't know what the javascript code convention is, so please correct me if i'm wrong. It would be nicer if we broke up this code over several lines, so that it's easier to read/digest. Something like
```
(
  function(i,s,o,g,r,a,m){
    i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();
      a=s.createElement(o), m=s.getElementsByTagName(o)[0];
      a.async=1;
      a.src=g;
      m.parentNode.insertBefore(a,m)
  }
)(window,document,'script','//www.google-analytics.com/analytics.js','ga');
```

Reply:
Perhaps also a comment on what the parameters `i, s, o, g, r, a, m` all stand for?

Again, I'm not a javascript expert. So if it's obvious to a javascript dev, then you can just ignore this comment

Reply:
See above. No need to "beautify" a snippet provided by Google. 

In a production scenario, you would transcompile (or at leasy minify) JS for the complete page, but this is a simple RtD page, and we're adding GA tracking code.